drivers/idle/ndl/ndl.c

```diff
@@ -4,6 +4,8 @@
  * Author: Artem Bityutskiy <artem.bityutskiy@linux.intel.com>
```

Comment:
wult: drivers: ndl: use pr_err to output error message

Cherry-picked this patch.